ovirt/resource_ovirt_vm.go

```diff
@@ -497,6 +532,33 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 	return resourceOvirtVMRead(d, meta)
 }
 
+func ovirtRemoveGraphicsConsoles(vmID string, meta interface{}) error {
+	conn := meta.(*ovirtsdk4.Connection)
+	vmGraphicConsoleService := conn.SystemService().VmsService().VmService(vmID).GraphicsConsolesService()
+
+	graphics, err := vmGraphicConsoleService.List().Send()
+	if err != nil {
+		if _, ok := err.(*ovirtsdk4.NotFoundError); ok {
+			return nil
+		}
+		return fmt.Errorf("error getting VM (%s) graphic consoles before deleting: %s", vmID, err)
+	}
+
+	for _, device := range graphics.MustConsoles().Slice() {
```

Comment:
When you remove one-by-one you can get to a situation of failure in the middle of the operation resulted in inconsistent state. You should rebuild and set the console list at once

Reply:
Do you mean remove them at once? We don't support it in ovirt API. Or you mean I should create new graphic consoles that were removed(each will require API call)?